tsconfig.json

```diff
@@ -35,6 +35,15 @@
     },
     {
       "path": "packages/gateway"
+    },
+    {
+      "path": "packages/datasource"
```

Comment:
Don't want to sound too pedantic, but shall we name it "datasources" and not "datasource"? :)

Reply:
`+`, renamed